src/style.css

```
@@ -0,0 +1,24 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+.main_block {
+  margin: 0;
+  font-size: 17px;
+  font-weight: bold;
+}
+
+p {
+  margin: 0;
+}
+
+.blocks {
```

```suggestion
p {
```

You may simply use `paragraph` selector for this task